### ISSUE TITLE
DEP und Umsatzzähler bei Rechnung

### ISSUE THREAD

**Issue by BalionK** [a-sit-plus/at-registrierkassen-mustercode#543]
Hallo!
Folgender Verkaufsvorgang findet an der PC-Kasse statt:
Verkauf einer Hose für € 159,- (Steuer 20 %) und einer Flasche Wein für € 10,- (Steuer 10 %). Gesamtpreis von € 169,- wird wie folgt bezahlt: € 20,- Gutscheineinlösung, € 49,- Barzahlung - Ausstellung einer Rechnung über den Restbetrag von € 100,- (Rechnung wird dann später vielleicht Bar oder per ec-Karte an der Kasse bezahlt - vielleicht aber auch überwiesen).
Was und welche Beträge müssen ins DEP und in den Umsatzzähler? Rechnung ist eigentlich kein Barumsatz. Kann trotzdem der Gesamtumsatz von € 169,- mit den entsprechenden Steuersätzen ins DEP und in den Umsatzzähler?
Vielen Dank schon mal für die Antworten!

**Comment by Flanelli** [comment]
Wenn ohnehin schon  feststeht dass die Rechnung nikcht zur Gänze bezahlt wird (  und eventuell sogar noch eine spätere Bezahlung per Banküberweisung statfinden könnte ) dann ist der einfachst Weg ohne irgendwleche Gedanken an aliquote Aufteilungen der Teilzahlungen meines Erachtens wie folgt:
Ausstellen einer Rechnung mit gesamten betrag offen ( also formal eine Zielrechnung )
Dieser Rechnung wird dem Rechnungslegungsgesetz entsprechend erstellt und hat somit auch automatisch alle erforderlichen Angaben über Leistungen, Artikel, Beträge, Umsatstzsuer ect. inne,
Diese Rechnung wird NICHT signiert da dies nicht zeingend sein muß.
Danach wird ein Zahlungsbeleg erstellt mit den bon iÍhnen angefphrten Zahkungsarten oder auch allen anderen noch erdenklichen Splittings wie Gutschrifteinlösung und kredikartenzahlung.
Dieser Beleg MUSS signiert werden aber der hesamte Zahkbetrag wird im Stzeuersatz NULL% abgelegt,
Am Beleg erfolgt ein Andruck mkit einem klaren Hiwneism auf die zughrunde liegende Rechnung, also
am besten Rechnungsnummer und Datum als referenz andrucken.
Somit ist keine aliqote Darstellung der verschiednen STuersätze erforderlich.
Kommt der Kunde später niochmals einen Teilbetrag ( oder den Rest ) an der Kasse zahlen wird wieder ein solcher SIGNIERTER Zahlungsbeleg gedruckt.
Bezahlt der Kunde den Rest per Banküberweisung ist das auch nur ein buchhalterishcer Vorgang und berührt die RKSV nicht.

**Comment by BalionK** [comment]
Vielen Dank!
Aber es wäre auch nicht schlimm den kompletten Vorgang freiwillig ins DEP und den Umsatzzähler aufzunehmen? Sollte ja eigentlich ein Bar-Umsatzzähler sein - ist er aber auch nicht wirklich, da Ausgaben ja auch nicht reinmüssen.

**Comment by Flanelli** [comment]
Schlimm wäre es natürlich nicht - letztendlich bleibt es halt "Geschmacksache"

**Comment by BalionK** [comment]
Wichtig für mich ist, dass es kein Verstoß gegen das Gesetz ist (und damit auch keine Strafen nach sich zieht für Anwender und Kassenhersteller), wenn Nicht-Bar-Umsätze ins DEP und den Umsatzzähler kommen

**Comment by Flanelli** [comment]
Lt. RKSV dürfen Sie alles signieren solange der Signiervorgang mit allem Drum und Dran bei jedem Beleg
den Vorgaben entspricht. Sie könnten daher auch theoretisch diverse Bargeld-Ein-und Ausgänge 
wie Geldentnahme zur Bank, Portozahlungen, Blumengesteck für Audlage eingekauft, Geldeinlage in Kassa
und alle nur erdenklichen Durchlaufposten freiwillig signieren. Ihr Problem besteht nur darin alle signierten Belege-sprich Umsätze dem jeweils richtigen Betrags-Satz zuzuordnen und da wird es dann eben doch
fallweise haarig wenn das nicht ganz exakt durchdacht automatisiert ist.

**Comment by ciscllc** [comment]
Hat die Flasche Wein nicht 20%?

**Comment by Flanelli** [comment]
@cislic Darum geht es zwar nicht im Grundsatz der Frage von BalionK aber es sind auch nicht in jedem Falle
20% anzuwenden. Bei ab Hof-Verkauf würde 13% zum Tragen kommen und noch kniffliger wird es bei
den sogenannten Restaurantumsätzen mit begünstigten Angeboten wie z.B.: Komplettmenüs.
Sind zwar in Österreich nicht gerade tagüblich aber umsatzsteuerrechtlich zumindest ein Sonderfall.

**Comment by ciscllc** [comment]
Ja, aber ich bezweifle, dass das in diesem Fall ein "nicht 20%" MwSt. Fall ist. Außer, es handelt sich um einen Bauern, der ein Fastfoodlokal unterhält, in dem er Menüs verkauft, die nur Wein enthalten und dazu Hosen...
Aber wenn es EXTRA ausgewiesen wird, sollte es ihm auch mitgeteilt werden, das hier ein Fehler besteht.
